lib/assets/stylesheets/overrides.scss

```diff
@@ -262,6 +262,16 @@ $header-background: linear-gradient(#ED6406, #F59503);
     height: 102px;
     padding-top: 0;
   }
+  #org-navbar .org-links {
+    padding-top: 15px;
+  }
+  #org-navbar .org-a {
+    line-height: 36px !important;
```

Comment:
prefer not to use `!important` as it essentially the nature/benefit of a cascading stylesheet. You should be able to override the default Bootstrap behavior by being more specific with your selector. 
Something like `#org-navbar .navbar-nav li .org-a` should get you what you want